src/libsystemd/sd-event/sd-event.c

```diff
@@ -37,6 +37,14 @@ static bool EVENT_SOURCE_WATCH_PIDFD(sd_event_source *s) {
                 s->child.options == WEXITED;
 }
 
+static bool EVENT_SOURCE_IS_ONLINE(sd_event_source *s) {
```

Comment:
Why is this uppercase? This is not a macro but a normal inline function.

Reply:
it's macro-like though?

will change